docs/core/policies.rst

```diff
@@ -197,56 +197,59 @@ set the ``random_seed`` attribute of the ``KerasPolicy`` to any integer.
 Embedding Policy
 ^^^^^^^^^^^^^^^^
 
-Transformer Embedding Dialogue Policy (TEDP)
+    .. warning::
+
+        ``EmbeddingPolicy`` got renamed to ``TEDPolicy``. Please use :ref:`ted_policy` instead of ``EmbeddingPolicy``.
+        The functionality of the policy stayed the same.
+
+
+.. _ted_policy:
 
-Transformer version of the Recurrent Embedding Dialogue Policy (REDP)
-used in our paper: `<https://arxiv.org/abs/1811.11707>`_
+TED Policy
+^^^^^^^^^^
+
+Transformer Embedding Dialogue (TED) Policy
+
+The policy used in our paper https://arxiv.org/abs/1910.00486.
 
 This policy has a pre-defined architecture, which comprises the
 following steps:
 
-    - concatenate user input (user intent and entities),
-      previous system action, slots and active form
-      for each time step into an input vector
-      to pre-transformer embedding layer;
+    - concatenate user input (user intent and entities), previous system action, slots and active form for each time
+      step into an input vector to pre-transformer embedding layer;
```

Comment:
can we keep these explanations a bit more high level? If somebody wants to know the details they should refer to the paper in my opinion.

Reply:
will be part of a separate PR https://github.com/RasaHQ/rasa/issues/5308